src/sentry/workflow_engine/migration_helpers/utils.py

```diff
@@ -20,13 +22,26 @@ def get_action_description(action: AlertRuleTriggerAction) -> str:
     """
 
     if action.type == AlertRuleTriggerAction.Type.EMAIL.value:
-        if action.target:
-            if action.target_type == AlertRuleTriggerAction.TargetType.USER.value:
-                action_target_user = cast(RpcUser, action.target)
-                return "Email " + action_target_user.email
-            elif action.target_type == AlertRuleTriggerAction.TargetType.TEAM.value:
-                action_target_team = cast(Team, action.target)
-                return "Email #" + action_target_team.slug
+        if action.target_type == AlertRuleTriggerAction.TargetType.USER.value:
+            try:
+                org_member = OrganizationMember.objects.get(
+                    user_id=action.target_identifier,
+                    organization=action.alert_rule_trigger.alert_rule.organization,
+                )
+            except OrganizationMember.DoesNotExist:
+                logger.info(
+                    "Organization member not found", extra={"user_id": action.target_identifier}
+                )
+                return "Email missing user"
```

Comment:
I am not entirely sure what happens if you have a rule set to email a specific user or team and they no longer exist, but I figure this shouldn't stop dual writing so we'll just log it and put a placeholder value in.